src/cursor/cur_hs.c

```diff
@@ -141,3 +141,100 @@ __wt_hs_cursor_search_near(WT_SESSION_IMPL *session, WT_CURSOR *cursor, int *exa
       session, WT_ISO_READ_UNCOMMITTED, ret = cursor->search_near(cursor, exactp));
     return (ret);
 }
+
+/*
+ * __curhs_close --
+ *     WT_CURSOR->close method for the hs cursor type.
+ */
+static int
+__curhs_close(WT_CURSOR *cursor)
+{
+    WT_CURSOR *file_cursor;
+    WT_CURSOR_HS *hsc;
```

Comment:
I feel like name `hsc` is not very intuitive. Maybe change it to `hs_cursor`?

Same comment for `__curhs_reset` and `__wt_curhs_open`

Reply:
I'm following the naming convention of other cursor types. I agree hsc is a bit short. I'll change that. I think for the function naming we need to be consistent for now.

Reply:
I meant to say that the same variable name has been used in `__curhs_reset` and `__wt_curhs_open`. I have no objection on the function naming - sorry for the confusion.